core/server/data/schema/default-settings.json

```diff
@@ -118,5 +118,16 @@
         "public_hash": {
             "defaultValue": null
         }
+    },
+    "members": {
+        "members_public_key": {
```

Comment:
We don't actually use or need this - as it can be derived from the private key - wondering if we should even bother storing it?